Dockerfile

```
@@ -0,0 +1,11 @@
+FROM islasgeci/jupyter:ff82
+# Instala paquetes de R
+RUN conda install --quiet --yes \
+    'r-car' \
+    'r-emmeans' \
+    'r-ggpubr' \
+    'r-lme4' \
+    'r-mumin' \ 
+        && \
+    conda clean --all -f -y && \
```

por favor agrega la versión larga:
```suggestion
    conda clean --all -f --yes && \
```
No encontré la versión larga para `-f`.